### ISSUE TITLE
OpenType.js API signature changed/incompatible + OpenType.js latest release

### ISSUE THREAD

**Issue by strarsis** [bitinn/character-map#1]
The  version of opentype.js package listed in this package 
as a dependency (^0.4.7 => v0.4.11) uses a different API,
the expected returned structure is different now.

```
npm install -g character map
character-map -f ./test.ttf
```

```
[...]/node_modules/character-map/index.js:29
        font.glyphs.forEach(function(glyph) {
                    ^

TypeError: font.glyphs.forEach is not a function
    at [...]/node_modules/character-map/index.js:29:14
    at [...]/node_modules/character-map/node_modules/opentype.js/src/opentype.js:220:16
    at [...]/node_modules/character-map/node_modules/opentype.js/src/opentype.js:52:9
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:380:3)
```

You can find the fixed code here: https://github.com/strarsis/font-chars/blob/master/index.js
It may be helpful to use font-chars internally in this CLI tool directly.

Furthermore opentype.js implemented WOFF support now, since release v0.5.0, 
so upgrading the opentype.js dependency would add WOFF support to this tool, too.

**Comment by tderkert** [comment]
Just had the same issue. 
@strarsis: Is there a simple solution to this?

**Comment by strarsis** [comment]
@tderkert: Yes, I based off an own module with fixed API usage:
https://www.npmjs.com/package/font-chars
See that module README for an example.
With a command line argument module (e.g. args) it should be fine as a cli tool, too.

**Comment by tderkert** [comment]
@strarsis Did a pull request with a working fix similar to yours. Your for loop looks prettier though so maybe i should update that and make another pull request.

**Comment by bitinn** [comment]
released as v1.1.0, thx for the PR
